gfxr_ext/decode/dive_annotation_processor_test.cpp

```diff
@@ -169,5 +169,255 @@ TEST(WriteBlockEndTest, MultipleSubmitsWithCommandsCreatesSubmitsWithCorrectComm
     EXPECT_THAT(vk_commands_cache[1002][2],
                 VulkanCommandInfoEqual(cmd_data_7.GetFunctionName(), 0, args_cmd_7));
 }
+
+TEST(WriteBlockEndTest,
+     SingleSubmitWithSingleCommandBufferHasCorrectNumberOfDrawCallsPerCommandBufferRecord)
+{
+    DiveAnnotationProcessor processor;
+
+    // Command Buffer 1 (handle 1001)
+    nlohmann::ordered_json           args_cmd_1 = { { "commandBuffer", 1001 } };
```

Comment:
i feel it could be cleaner if we have a helper function like:
```
gfxrecon::util::DiveFunctionData CreateCommandData(const std::string& name,
                                                   uint64_t command_buffer_handle,
                                                   uint32_t cmd_buffer_index,
                                                   uint32_t block_index)
{
    nlohmann::ordered_json args = { { "commandBuffer", command_buffer_handle } };
    return gfxrecon::util::DiveFunctionData(name,
                                            cmd_buffer_index,
                                            block_index,
                                            args);
}
```

Reply:
Added.